src/app/list-manager/list-manager.component.ts

```diff
@@ -19,6 +19,7 @@ import {
       transition(':enter,:leave', [animate('0.5s')]),
     ]),
   ],
+  providers: [StoreServiceService],
```

Comment:
I would provide this at the highest level that you'd reasonably want to access the service. This would make it so that you'd have to be a child of the `ListManagerComponent` in order to access the service. 
- [ ] Move this provider up to `AppComponent` or to the `AppModule`